quiche/src/lib.rs

```diff
@@ -4805,6 +4806,44 @@ impl Connection {
         MIN_CLIENT_INITIAL_LEN
     }
 
+    /// Schedule an ack-eliciting packet on the active path.
+    ///
+    /// QUIC packets might not contain ack-eliciting frames during normal
+    /// operating conditions. If the packet would already contain
+    /// ack-eliciting frames, this method does not change any behavior.
+    /// However, if the packet would not ordinarily contain ack-eliciting
+    /// frames, this method ensures that a PING frame sent.
+    ///
+    /// Calling this method multiple times before [`send()`] has no effect.
```

Comment:
This should fix the warning

```suggestion
    /// Calling this method multiple times before [`send()`] has no effect.
    
    /// [`send()`]: struct.Connection.html#method.send
```